app/Console/Command/DeletePlace.php

```diff
@@ -0,0 +1,120 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Console\Command;
+
+use Broadway\CommandHandling\CommandBus;
+use CultuurNet\UDB3\Event\Commands\UpdateLocation;
+use CultuurNet\UDB3\Event\ReadModel\Relations\EventRelationsRepository;
+use CultuurNet\UDB3\Event\ValueObjects\LocationId;
+use CultuurNet\UDB3\Offer\Commands\DeleteOffer;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Question\ConfirmationQuestion;
+
+/**
+ * @url https://jira.publiq.be/browse/III-6109
+ */
+class DeletePlace extends AbstractCommand
+{
+    private const FORCE = 'force';
+    private const DRY_RUN = 'dry-run';
+    private const PLACE_UUID = 'place-uuid';
+    private const CANONICAL_UUID = 'canonical-uuid';
+
+    private EventRelationsRepository $eventRelationsRepository;
+
+    public function __construct(
+        CommandBus $commandBus,
+        EventRelationsRepository $eventRelationsRepository
+    ) {
+        parent::__construct($commandBus);
+        $this->eventRelationsRepository = $eventRelationsRepository;
+    }
+
+    public function configure(): void
+    {
+        $this
+            ->setName('place:delete')
+            ->setDescription('Mark one place as deleted, and move all it\'s events to the canonical place')
+            ->addArgument(
+                self::PLACE_UUID,
+                null,
+                'Place uuid to delete.'
+            )
+            ->addArgument(
+                self::CANONICAL_UUID,
+                null,
+                'Canonical place uuid to move all events towards.'
+            )
+            ->addOption(
```

Comment:
A `force` option is generally only needed, when it will be run via Jenkins by the helpdesk colleagues.
Since this is a one-time `Command` I do not think a `force` option is needed.